app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    f_in = command.split(" ")[1]
+    f_out = command.split(" ")[2]
```

Comment:
Consider using more descriptive variable names instead of `f_in` and `f_out`, as suggested in the checklist: 'Use descriptive and correct variable names.'